### ISSUE TITLE
Build environment with pip only (remove conda)

### ISSUE THREAD

**Issue by khider** [LinkedEarth/Pyleoclim_util#463]
Have the environment build through pip for CI and doc.

For doc, also need to specify the base libraries since they are no longer included in the base envt for readthedocs.

**Comment by jordanplanders** [comment]
I have tested building the environment needed to support  PyleoTutorials from a requirements.txt file in the paleobook_templates repo (as a test bed) which can be found in built form here: https://mybinder.org/v2/gh/LinkedEarth/paleobook_templates/main

It looks like if we modify the requirements.txt in doc_build to include all of the sphinx stuff, and modify the .readthedocs.yml file to point to that requirements file, we should be in business. Additionally, we can modify the requirements.txt to take over the heavy lifting from environment.yml (this set should be a modified subset of the set I checked with my binder, i.e. a set that doesn't include Pyleoclim & Pylipd).
